src/proto/nc_redis.c

```diff
@@ -2787,6 +2807,9 @@ redis_post_coalesce(struct msg *r)
     case MSG_REQ_REDIS_DEL:
         return redis_post_coalesce_del(r);
 
+    case MSG_REQ_REDIS_UNLINK:
+        return redis_post_coalesce_unlink(r);
```

Comment:
I think this could just call redis_post_coalesce_del if the method implementations  are the same (e.g. add a comment that the method is used for DEL or UNLINK)